packages/web/components/ui/column-node.tsx

```
@@ -221,90 +223,90 @@ function ColumnFloatingToolbar({ children }: React.PropsWithChildren) {
-    fill="none"
-    height="16"
-    viewBox="0 0 16 16"
-    width="16"
-    xmlns="http://www.w3.org/2000/svg"
+    fill='none'
+    height='16'
+    viewBox='0 0 16 16'
+    width='16'
+    xmlns='http://www.w3.org/2000/svg'
     {...props}
   >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)